utils/python/CIME/case.py

```diff
@@ -68,18 +68,20 @@ def get_value(self, item, attribute={}, resolved=True, subgroup=None):
         logging.info("Not able to retreive value for item '%s'" % item)
 
 
-    def get_resolved_value(self, item):
```

Comment:
This may conflict with what I just pushed. Please rewrite history to remove any changes to this file.